llarp/router/router.cpp

```diff
@@ -1402,8 +1407,10 @@ namespace llarp
   void
   Router::QueueWork(std::function<void(void)> func)
   {
-    _loop->call_soon(func);
-    // m_lmq->job(std::move(func));
+    if (m_isServiceNode)
```

Comment:
could stand to have a comment here clarifying the two cases and why the decision to handle client/relay differently.